### PR TITLE
Mount menu entry into Vue user menu

### PR DESCRIPTION
**Pull request by Pytal**
- For https://github.com/nextcloud/server/pull/36232

### Requires

- [x] https://github.com/nextcloud/firstrunwizard/pull/815
- ~https://github.com/nextcloud/server/pull/36449~
- [x] https://github.com/nextcloud/server/pull/36508

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -37,7 +37,7 @@ The First run wizard can be customized to meet specific design goals, or to chan
 		<navigation>
 			<id>firstrunwizard_about</id>
 			<name>About</name>
-			<route>files.view.index</route>
+			<route />
```
Comment (nickvergessen) [inline appinfo/info.xml:40]: This looks wrong?! Now the settings menu entry does not open for me anymore.

Comment (nickvergessen) [reply appinfo/info.xml:40]: lint info.,xml agrees:
```
 ./appinfo/info.xml:40: element route: Schemas validity error : Element 'route': [facet 'minLength'] The value has a length of '0'; this underruns the allowed minimum length of '1'.
./appinfo/info.xml:40: element route: Schemas validity error : Element 'route': [facet 'pattern'] The value '' is not accepted by the pattern '[0-9a-zA-Z_]+(\.[0-9a-zA-Z_]+){2}'.
./appinfo/info.xml:40: element route: Schemas validity error : Element 'route': '' is not a valid value of the atomic type 'route'.
./appinfo/info.xml fails to validate
```

Comment (Pytal) [reply appinfo/info.xml]: Fixed

This is only meant to work on top of https://github.com/nextcloud/firstrunwizard/pull/815 and https://github.com/nextcloud/server/pull/36232

**Comment by szaimen** [comment]
See https://github.com/nextcloud/server/pull/36232#issuecomment-1407188075

File: appinfo/info.xml (hunk 1 above)
Comment (Pytal) [reply appinfo/info.xml]: `xmllint` schema updated in https://github.com/nextcloud/server/pull/36449

Comment (Pytal) [reply appinfo/info.xml]: `xmllint` schema updated to allow no route in https://github.com/nextcloud/server/pull/36508

**Comment by Pytal** [comment]
Rebased

Lint is now passing https://github.com/nextcloud/firstrunwizard/actions/runs/4088986130/jobs/7051216647 with schema update in https://github.com/nextcloud/server/pull/36508

**Review by JuliaKirschenheuter — APPROVED** [review]
can't reproduce https://github.com/nextcloud/server/pull/36232#issuecomment-1407188075, but probably have no case

**Comment by Pytal** [comment]
Merge after https://github.com/nextcloud/server/pull/36232

**Review by szaimen — APPROVED** [review]
Tested together with https://github.com/nextcloud/server/pull/36232 and works now but didnt review the code.

(Looks like the internal server issue was somehow caused by the viewer app not being installed).

**Pull request merged**